### PR TITLE
Thread backfilling endpoints

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim let's get this party started.

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/Keep.scala [hunk 1]
```diff
@@ -305,6 +305,7 @@ object KeepSource {
   val twitterFileImport = KeepSource("twitterFileImport")
   val twitterSync = KeepSource("twitterSync")
   val slack = KeepSource("slack")
+  val backfilledThread = KeepSource("backfilledThread")
```
Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala]: Not sure about that one, that we want to have to carry that around forever.

Comment (leogrim) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala]: You can use `MessageSource` to properly backfill this.

Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala]: How about `discussion`?

Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala]: I like it.

**Pull request merged**